spec/routes/api/project/location/postgres_spec.rb

```diff
@@ -11,7 +11,7 @@
     Prog::Postgres::PostgresResourceNexus.assemble(
       project_id: project.id,
       location: "hetzner-fsn1",
-      name: "pg-with-permission",
+      name: "postgres-with-permission",
```

Comment:
This doesn't appear related to the other changes.

Reply:
I thought that with the changes, things shouldn't have names that start with the same prefix as their id would. I'm pretty sure this was throwing errors once I made the other changes. So it seemed related, am I missing something in my understanding?

Reply:
Ubids cannot have dashes, and this isn't exactly 26 characters, so this isn't ambiguous.  Only names that could be valid ubids for the model are rejected.

Reply:
Oof, right. I forgot how narrow the conflict was.